apps-rendering/src/client/article.ts

```diff
@@ -87,34 +105,116 @@ function followToggle(topic: Topic): void {
 	});
 }
 
-function topicClick(e: Event): void {
-	const follow = document.querySelector('.js-follow');
+function notificationsFollowClick(e: Event): void {
+	const follow = document.querySelector('.js-follow-notifications');
 	const topic = getTopic(follow);
+	if (topic) {
+		followToggle(
+			topic,
+			'.js-follow-notifications-status',
+			FollowNotificationStatus,
+			notificationsClient,
+		);
+	}
+}
 
+function tagFollowClick(e: Event): void {
+	const follow = document.querySelector('.js-follow-tag');
+	const topic = getTopic(follow);
 	if (topic) {
-		followToggle(topic);
+		followToggle(
+			topic,
+			'.js-follow-tag-status',
+			FollowTagStatus,
+			tagClient,
+		);
 	}
 }
 
+function conditionallyRenderFollowTagComponent(
```

Comment:
just to keep names aligned, I think we could rename this function to `conditionallyRenderTagFollowComponent` as we've called the function above `tagFollowClick` -- assuming they're referring to the same thing!